math/pi/montecarlopi.go

```diff
@@ -25,3 +27,67 @@ func MonteCarloPi(randomPoints int) float64 {
 	pi := float64(inside) / float64(randomPoints) * 4
 	return pi
 }
+
+// MonteCarloPi2 approximates the value of pi using the Monte Carlo method.
+// Unlike the MonteCarloPi function (first version), this implementation uses
+// goroutines and channels to parallelize the computation.
+// More details on the Monte Carlo method available at https://en.wikipedia.org/wiki/Monte_Carlo_method.
+// More details on goroutines parallelization available at https://go.dev/doc/effective_go#parallel.
+func MonteCarloPi2(n int) (float64, error) {
```

Comment:
Maybe use
```suggestion
func MonteCarloPiConcurrent(n int) (float64, error) {
```

or 

```suggestion
func ConcurrentMonteCarloPi(n int) (float64, error) {
```
??

Reply:
Understood! Fixed in https://github.com/TheAlgorithms/Go/pull/448/commits/02e905a7f598ea23aa40171aae5ef9a6d72e03da